src/line-chart/app_1.css

```diff
@@ -0,0 +1,281 @@
+@font-face {
+	font-family: quicksand-light;
```

Comment:
One space indent. Do we want 2 or 4? IMO one space is a bit hard to read generally.